transport-native-unix-common/src/main/c/netty_unix_socket.c

```diff
@@ -183,6 +189,27 @@ static int socket_type(JNIEnv* env) {
         }
         return AF_INET6;
     } else {
+        int optval = 1;
+        if (netty_unix_socket_setOption0(fd, IPPROTO_IPV6, IPV6_V6ONLY, &optval, sizeof(optval)) < 0) {
```

Comment:
Oh, if its only called once thats fine.  I thought it was called for each socket.  NM.